scenarios.py

```diff
@@ -303,13 +403,14 @@ def reset(self, *, seed=None, options=None):
         self.reset_state("intensity_all_regions")
         self.reset_state("mitigation_rates_all_regions")
 
-        # additional climate states for carbon model
+        # additional climate states for carbon and temperature model
         self.reset_state("global_alpha")
         self.reset_state("global_carbon_reservoirs")
         self.reset_state("global_cumulative_emissions")
         self.reset_state("global_cumulative_land_emissions")
         self.reset_state("global_emissions")
         self.reset_state("global_acc_pert_carb_stock")
+        self.reset_state('global_temperature_boxes')
```

Comment:
What is the definition of this one?

Reply:
I have no idea but it was added to the main rice class, maybe something from Fair? It wouldn’t run without it. Could be that someone committed directly to gaia_merge adding this observation. I added it bc otherwise wasn’t compatible w base rice.